src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -520,29 +520,54 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
         OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::intel_cpu_LT, "LowPrecisionTransformations");
-
-        auto supportedPrecisions = std::vector<PrecisionsRestriction>({
-            PrecisionsRestriction::create<ngraph::opset1::Convolution>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}},
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}}
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
-                {0, {ngraph::element::u8}},
-                {1, {ngraph::element::i8}}
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::Multiply>({
-                {0, {ngraph::element::u8}},
-                {1, {ngraph::element::i8}},
-            }),
-            PrecisionsRestriction::create<ngraph::opset1::MatMul>({
-                {0, {ngraph::element::u8, ngraph::element::i8}},
-                {1, {ngraph::element::i8}}
-            }),
-        });
+        std::vector<PrecisionsRestriction> supportedPrecisions{};
+        //Only enable conv/group conv signed input on AMX platform.
+        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx))
+            supportedPrecisions = std::vector<PrecisionsRestriction>({
+                PrecisionsRestriction::create<ngraph::opset1::Convolution>({
+                    {0, {ngraph::element::u8, ngraph::element::i8}},
+                    {1, {ngraph::element::i8}},
+                }),
+                PrecisionsRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
+                    {0, {ngraph::element::u8, ngraph::element::i8}},
+                    {1, {ngraph::element::i8}}
+                }),
+                PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
+                    {0, {ngraph::element::u8, ngraph::element::i8}},
+                    {1, {ngraph::element::i8}}
+                }),
+                PrecisionsRestriction::create<ngraph::opset1::Multiply>({
+                    {0, {ngraph::element::u8}},
+                    {1, {ngraph::element::i8}},
+                }),
+                PrecisionsRestriction::create<ngraph::opset1::MatMul>({
+                    {0, {ngraph::element::u8, ngraph::element::i8}},
+                    {1, {ngraph::element::i8}}
+                }),
+            });
+        else
+            supportedPrecisions = std::vector<PrecisionsRestriction>({
+                PrecisionsRestriction::create<ngraph::opset1::Convolution>({
+                    {0, {ngraph::element::u8}},
+                    {1, {ngraph::element::i8}},
+                }),
+                PrecisionsRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
+                    {0, {ngraph::element::u8, ngraph::element::i8}},
```

Comment:
ConvolutionBackpropData has supported the I8/U8 input precision in 2022.1 release. And why we disabled I8 input for avx512-vnni is that onednn I8 input compensation implement (2022.2 introduced) perf is lower than U8 OV legacy input zero point(2022.1).  As for  ConvolutionBackpropData, First we don't have legacy input zero point mechanism in OV and we should keep same config with 2022.1 to avoid regression. So I think we should keep support I8 input precision for ConvolutionBackpropData as 2022.1. Do you think it is okay?